lib/manageiq/appliance_console/message_configuration_server.rb

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
why this newline?

Reply:
Leftover from another change I pulled out of this PR